app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
```

Comment:
- [ ] Why returning @post in comments new. Kindly remove it

Reply:
I wanted using the `@post.id` for routing. I later didn't us it so it is actually redundant as you have observed. Thanks @deepakdanger for the recommendation!